test/torchaudio_unittest/models/decoder/cuda_ctc_decoder_test.py

```diff
@@ -0,0 +1,49 @@
+import torch
+from parameterized import parameterized
+from torchaudio_unittest.common_utils import (
+    get_asset_path,
+    skipIfNoCuCtcDecoder,
+    skipIfNoCuda,
+    TempDirMixin,
+    TorchaudioTestCase,
+)
+
+NUM_TOKENS = 7
+
+
+@skipIfNoCuda
+@skipIfNoCuCtcDecoder
+class CUCTCDecoderTest(TempDirMixin, TorchaudioTestCase):
+    def _get_decoder(self, tokens=None, **kwargs):
+        from torchaudio.models.decoder import cuda_ctc_decoder
+
+        if tokens is None:
+            tokens = get_asset_path("decoder/tokens.txt")
+
+        return cuda_ctc_decoder(
+            tokens=tokens,
+            beam_size=5,
+            **kwargs,
+        )
+
+    def _get_emissions(self):
+        B, T, N = 4, 15, NUM_TOKENS
+
+        emissions = torch.rand(B, T, N).cuda()
+        emissions = torch.nn.functional.log_softmax(emissions, -1)
+
+        return emissions
+
+    @parameterized.expand(
+        [(get_asset_path("decoder/tokens.txt"),), (["-", "|", "f", "o", "b", "a", "r"],)],
```

Comment:
The part `(get_asset_path("decoder/tokens.txt")` could cause an issue, as the full path is appended to test name.

Can we split these tests into two different methods? like
`test_construct_basic_decoder_path`, `test_construct_basic_decoder_tokens`